### PR TITLE
CP-928 Use dart_dev for tooling, w_service

### PR DESCRIPTION
**Pull request by evanweible-wf**
## Issue
Dart_dev is available and should be used for tooling.

## Changes
**Source:**
- Dart formatter run

**Tooling:**
- dart_dev used now for static analysis, tests, code coverage, serving examples, and code formatting.

**Tests:**
- n/a

## TODO
- [x] Update license headers, make sure license is applied to all files
- [x] Update readme to include dev instructions

## Areas of Regression
- Tooling

## Testing
- Ensure the tooling works as expected.

## Code Review
@trentgrover-wf
@maxwellpeterson-wf 
@dustinlessard-wf
@jayudey-wf

### CODE REVIEW

**Comment by maxwellpeterson-wf** [comment]
Looks like there are some test failures

**Comment by evanweible-wf** [comment]
I have no idea why those tests are failing.. seems as though content shell is failing to load. Tests pass locally, I'll keep looking into it.

**Comment by trentgrover-wf** [comment]
+1, pending resolution of the test run failures

**Comment by evanweible-wf** [comment]
@trentgrover-wf @maxwellpeterson-wf @dustinlessard-wf CI checks are passing now.

File: README.md [hunk 1]
```diff
@@ -54,4 +54,10 @@ import 'package:w_service/w_service_server.dart' show configureWServiceForServer
 void main() {
   configureWServiceForServer();
 }
-```
\ No newline at end of file
+```
+
+## Development
+
+Truss leverages the dart_dev package for most of its tooling needs, including
```
Comment (trentgrover-wf) [inline README.md]: Truss?

Comment (evanweible-wf) [inline README.md]: You would think I'd learn

**Comment by maxwellpeterson-wf** [comment]
+1

**Comment by evanweible-wf** [comment]
fyi the inconsistent content-shell issues I'm seeing on these Travis CI builds seems to be this: https://github.com/travis-ci/travis-ci/issues/4406

**Comment by evanweible-wf** [comment]
@trentgrover-wf @maxwellpeterson-wf @dustinlessard-wf okay it really is ready now! I also snuck in the codecov and pub badges that were missing in the readme.

**Comment by maxwellpeterson-wf** [comment]
+1

**Comment by trentgrover-wf** [comment]
+1
@jayudey-wf ready for merge

**Comment by jayudey-wf** [comment]
QA Resource Approval: +10

- [ ] Testing instruction
- [ ] Dev +1's
- [ ] Dev/QA +10 with detail of what was tested
 - verified that tooling was working as expected (`test`, `analyze`, `format`,`examples`)
- [ ] Unit test created/updated
- [ ] All unit tests pass

Merging into master.

**Comment by jayudey-wf** [comment]
@rosie run_merge_script

**Pull request merged**